src/p_db.c

```diff
@@ -1071,7 +1071,8 @@ prim_set(PRIM_PROTOTYPE)
                 || string_prefix("autostart", flag)
                 || string_prefix("abate", flag))
             tmp = ABODE;
-        else if (string_prefix("builder", flag))
+        else if (string_prefix("builder", flag)
```

Comment:
Perhaps we should note in both this code and in has_flag that any change needs to be made in both places?

Ideally, we'd unify these 2 case statements in some way since it looks like this has a lot of overlap with has_flag's code.  However, there's also such a thing as optimizing yourself until you're weary, so if you rather put a TODO and a note that both places need to be changed, I wouldn't judge you :)

If you wanted to go the extra mile, though, I'd say make a str_to_flag call that has this case statement, then both has_flag and prim_set can use it (along with anywhere else this mess lives).

Reply:
str_to_flag is definitely coming - just wanted to see if I could be clever with ALL the flag checking. I can certainly add str_to_flag to this PR in the meantime so it definitely doesn't wait for a unified process.

Reply:
I think both changes are made. Please review; I think everything's in order...

I'm not too thrilled about the duplicate "truewizard" check, but it seemed like the best way (right now)? to preserver the different meanings for `SET` and `FLAG?`.

do_set is on notice as needing a change, too, but it conflates the str_to_flag logic with the restricted function (while introducing its own restrictions) and will take more thought.